libs/api/src/lib/memberContext.ts

```diff
@@ -694,14 +700,66 @@ export class MemberContext implements MemberContextInterface {
       throw new InternalError()
     }
 
-    const invoice = await this.renewSubscriptionForUser({subscription})
+    if (startsAt < now || paidUntil) {
```

Comment:
@elias-summermatter could this if/else statement break the special subscriptions at Hauptstadt (subscriptions already paid at the end of this year, but valid from March 2025 on) or the normal subscriptions at Tsüri or elsewhere?

How about adding one or two extra tests? Could it be worth it?

Reply:
Since it just impactes new subscriptions i would say no